### ISSUE TITLE
2025-05-19, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#174]
# Today's Arxiv Papers

(1) [SoftCoT++: Test-Time Scaling with Soft Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.11484v1), 2025-05-16

(2) [msf-CNN: Patch-based Multi-Stage Fusion with Convolutional Neural Networks for TinyML](http://arxiv.org/abs/2505.11483v1), 2025-05-16

(3) [Improving Assembly Code Performance with Large Language Models via Reinforcement Learning](http://arxiv.org/abs/2505.11480v1), 2025-05-16

(4) [HelpSteer3-Preference: Open Human-Annotated Preference Data across Diverse Tasks and Languages](http://arxiv.org/abs/2505.11475v1), 2025-05-16

(5) [Disentangling Reasoning and Knowledge in Medical Large Language Models](http://arxiv.org/abs/2505.11462v1), 2025-05-16

(6) [ProxyPrompt: Securing System Prompts against Prompt Extraction Attacks](http://arxiv.org/abs/2505.11459v1), 2025-05-16

(7) [LLMs unlock new paths to monetizing exploits](http://arxiv.org/abs/2505.11449v1), 2025-05-16

(8) [Is Compression Really Linear with Code Intelligence?](http://arxiv.org/abs/2505.11441v1), 2025-05-16

(9) [GODBench: A Benchmark for Multimodal Large Language Models in Video Comment Art](http://arxiv.org/abs/2505.11436v1), 2025-05-16

(10) [MegaScale-MoE: Large-Scale Communication-Efficient Training of Mixture-of-Experts Models in Production](http://arxiv.org/abs/2505.11432v1), 2025-05-16

(11) [When Thinking Fails: The Pitfalls of Reasoning for Instruction-Following in LLMs](http://arxiv.org/abs/2505.11423v1), 2025-05-16

(12) [EdgeWisePersona: A Dataset for On-Device User Profiling from Natural Language Interactions](http://arxiv.org/abs/2505.11417v1), 2025-05-16

(13) [MoE-CAP: Benchmarking Cost, Accuracy and Performance of Sparse Mixture-of-Experts Systems](http://arxiv.org/abs/2505.11415v1), 2025-05-16

(14) [CARES: Comprehensive Evaluation of Safety and Adversarial Robustness in Medical LLMs](http://arxiv.org/abs/2505.11413v1), 2025-05-16

(15) [Visual Planning: Let's Think Only with Images](http://arxiv.org/abs/2505.11409v1), 2025-05-16

(16) [Large Language Model Use Impact Locus of Control](http://arxiv.org/abs/2505.11406v1), 2025-05-16

(17) [EmotionHallucer: Evaluating Emotion Hallucinations in Multimodal Large Language Models](http://arxiv.org/abs/2505.11405v1), 2025-05-16

(18) [Can AI automatically analyze public opinion? A LLM agents-based agentic pipeline for timely public opinion analysis](http://arxiv.org/abs/2505.11401v1), 2025-05-16

(19) [GuideBench: Benchmarking Domain-Oriented Guideline Following for LLM Agents](http://arxiv.org/abs/2505.11368v1), 2025-05-16

(20) [Phare: A Safety Probe for Large Language Models](http://arxiv.org/abs/2505.11365v1), 2025-05-16

(21) [LegoSLM: Connecting LLM with Speech Encoder using CTC Posteriors](http://arxiv.org/abs/2505.11352v1), 2025-05-16

(22) [Context parroting: A simple but tough-to-beat baseline for foundation models in scientific machine learning](http://arxiv.org/abs/2505.11349v1), 2025-05-16

(23) [Benchmarking Critical Questions Generation: A Challenging Reasoning Task for Large Language Models](http://arxiv.org/abs/2505.11341v1), 2025-05-16

(24) [DecompileBench: A Comprehensive Benchmark for Evaluating Decompilers in Real-World Scenarios](http://arxiv.org/abs/2505.11340v1), 2025-05-16

(25) [XtraGPT: LLMs for Human-AI Collaboration on Controllable Academic Paper Revision](http://arxiv.org/abs/2505.11336v1), 2025-05-16

(26) [TokenWeave: Efficient Compute-Communication Overlap for Distributed LLM Inference](http://arxiv.org/abs/2505.11329v1), 2025-05-16

(27) [Search and Refine During Think: Autonomous Retrieval-Augmented Reasoning of LLMs](http://arxiv.org/abs/2505.11277v1), 2025-05-16

(28) [TCC-Bench: Benchmarking the Traditional Chinese Culture Understanding Capabilities of MLLMs](http://arxiv.org/abs/2505.11275v1), 2025-05-16

(29) [SelfBudgeter: Adaptive Token Allocation for Efficient LLM Reasoning](http://arxiv.org/abs/2505.11274v1), 2025-05-16

(30) [Semantic Caching of Contextual Summaries for Efficient Question-Answering with Language Models](http://arxiv.org/abs/2505.11271v1), 2025-05-16

(31) [TAIJI: MCP-based Multi-Modal Data Analytics on Data Lakes](http://arxiv.org/abs/2505.11270v1), 2025-05-16

(32) [DRAGON: A Large-Scale Dataset of Realistic Images Generated by Diffusion Models](http://arxiv.org/abs/2505.11257v1), 2025-05-16

(33) [LD-Scene: LLM-Guided Diffusion for Controllable Generation of Adversarial Safety-Critical Driving Scenarios](http://arxiv.org/abs/2505.11247v1), 2025-05-16

(34) [Concept Drift Guided LayerNorm Tuning for Efficient Multimodal Metaphor Identification](http://dx.doi.org/10.1145/3731715.3733296), 2025-05-16

(35) [Is PRM Necessary? Problem-Solving RL Implicitly Induces PRM Capability in LLMs](http://arxiv.org/abs/2505.11227v1), 2025-05-16

(36) [HAPO: Training Language Models to Reason Concisely via History-Aware Policy Optimization](http://arxiv.org/abs/2505.11225v1), 2025-05-16

(37) [Audio Turing Test: Benchmarking the Human-likeness of Large Language Model-based Text-to-Speech Systems in Chinese](http://arxiv.org/abs/2505.11200v1), 2025-05-16

(38) [Prot2Text-V2: Protein Function Prediction with Multimodal Contrastive Alignment](http://arxiv.org/abs/2505.11194v1), 2025-05-16

(39) [Can Global XAI Methods Reveal Injected Bias in LLMs? SHAP vs Rule Extraction vs RuleSHAP](http://arxiv.org/abs/2505.11189v1), 2025-05-16

(40) [On Next-Token Prediction in LLMs: How End Goals Determine the Consistency of Decoding Algorithms](http://arxiv.org/abs/2505.11183v1), 2025-05-16

(41) [Feasibility with Language Models for Open-World Compositional Zero-Shot Learning](http://arxiv.org/abs/2505.11181v1), 2025-05-16

(42) [mmRAG: A Modular Benchmark for Retrieval-Augmented Generation over Text, Tables, and Knowledge Graphs](http://arxiv.org/abs/2505.11180v1), 2025-05-16

(43) [Low-Resource Language Processing: An OCR-Driven Summarization and Translation Pipeline](http://arxiv.org/abs/2505.11177v1), 2025-05-16

(44) [From Intent Discovery to Recognition with Topic Modeling and Synthetic Data](http://arxiv.org/abs/2505.11176v1), 2025-05-16

(45) [Real-Time Verification of Embodied Reasoning for Generative Skill Acquisition](http://arxiv.org/abs/2505.11175v1), 2025-05-16

(46) [Gaussian Weight Sampling for Scalable, Efficient and Stable Pseudo-Quantization Training](http://arxiv.org/abs/2505.11170v1), 2025-05-16

(47) [SoLoPO: Unlocking Long-Context Capabilities in LLMs via Short-to-Long Preference Optimization](http://arxiv.org/abs/2505.11166v1), 2025-05-16

(48) [MPMA: Preference Manipulation Attack Against Model Context Protocol](http://arxiv.org/abs/2505.11154v1), 2025-05-16

(49) [Human-Aligned Bench: Fine-Grained Assessment of Reasoning Ability in MLLMs vs. Humans](http://arxiv.org/abs/2505.11141v1), 2025-05-16

(50) [Scaling Reasoning can Improve Factuality in Large Language Models](http://arxiv.org/abs/2505.11140v1), 2025-05-16

(51) [Navigating the Alpha Jungle: An LLM-Powered MCTS Framework for Formulaic Factor Mining](http://arxiv.org/abs/2505.11122v1), 2025-05-16

(52) [PARSEC: Preference Adaptation for Robotic Object Rearrangement from Scene Context](http://arxiv.org/abs/2505.11108v1), 2025-05-16

(53) [Group Think: Multiple Concurrent Reasoning Agents Collaborating at Token Level Granularity](http://arxiv.org/abs/2505.11107v1), 2025-05-16

(54) [Towards Better Evaluation for Generated Patent Claims](http://arxiv.org/abs/2505.11095v1), 2025-05-16

(55) [ShiQ: Bringing back Bellman to LLMs](http://arxiv.org/abs/2505.11081v1), 2025-05-16

(56) [BLEUBERI: BLEU is a surprisingly effective reward for instruction following](http://arxiv.org/abs/2505.11080v1), 2025-05-16

(57) [$\mathcal{A}LLM4ADD$: Unlocking the Capabilities of Audio Large Language Models for Audio Deepfake Detection](http://arxiv.org/abs/2505.11079v1), 2025-05-16

(58) [LLM-Enhanced Symbolic Control for Safety-Critical Applications](http://arxiv.org/abs/2505.11077v1), 2025-05-16

(59) [Addition is almost all you need: Compressing neural networks with double binary factorization](http://arxiv.org/abs/2505.11076v1), 2025-05-16

(60) [Time Travel is Cheating: Going Live with DeepFund for Real-Time Fund Investment Benchmarking](http://arxiv.org/abs/2505.11065v1), 2025-05-16

(61) [Think Twice Before You Act: Enhancing Agent Behavioral Safety with Thought Correction](http://arxiv.org/abs/2505.11063v1), 2025-05-16

(62) [OntoURL: A Benchmark for Evaluating Large Language Models on Symbolic Ontological Understanding, Reasoning and Learning](http://arxiv.org/abs/2505.11031v1), 2025-05-16

(63) [Logo-LLM: Local and Global Modeling with Large Language Models for Time Series Forecasting](http://arxiv.org/abs/2505.11017v1), 2025-05-16

(64) [WildDoc: How Far Are We from Achieving Comprehensive and Robust Document Understanding in the Wild?](http://arxiv.org/abs/2505.11015v1), 2025-05-16

(65) [Humans expect rationality and cooperation from LLM opponents in strategic games](http://arxiv.org/abs/2505.11011v1), 2025-05-16

(66) [Review-Instruct: A Review-Driven Multi-Turn Conversations Generation Method for Large Language Models](http://arxiv.org/abs/2505.11010v1), 2025-05-16

(67) [ReaCritic: Large Reasoning Transformer-based DRL Critic-model Scaling For Heterogeneous Networks](http://arxiv.org/abs/2505.10992v1), 2025-05-16

(68) [RAGSynth: Synthetic Data for Robust and Faithful RAG Component Optimization](http://arxiv.org/abs/2505.10989v1), 2025-05-16

(69) [Rethinking the Role of Prompting Strategies in LLM Test-Time Scaling: A Perspective of Probability Theory](http://arxiv.org/abs/2505.10981v1), 2025-05-16

(70) [Group-in-Group Policy Optimization for LLM Agent Training](http://arxiv.org/abs/2505.10978v1), 2025-05-16

(71) [Can Large Language Models Correctly Interpret Equations with Errors?](http://arxiv.org/abs/2505.10966v1), 2025-05-16

(72) [MPS-Prover: Advancing Stepwise Theorem Proving by Multi-Perspective Search and Data Curation](http://arxiv.org/abs/2505.10962v1), 2025-05-16

(73) [Let the Trial Begin: A Mock-Court Approach to Vulnerability Detection using LLM-Based Agents](http://arxiv.org/abs/2505.10961v1), 2025-05-16

(74) [SubGCache: Accelerating Graph-based RAG with Subgraph-level KV Cache](http://arxiv.org/abs/2505.10951v1), 2025-05-16

(75) [The Way We Prompt: Conceptual Blending, Neural Dynamics, and Prompt-Induced Transitions in LLMs](http://arxiv.org/abs/2505.10948v1), 2025-05-16

(76) [ToDMA: Large Model-Driven Token-Domain Multiple Access for Semantic Communications](http://arxiv.org/abs/2505.10946v1), 2025-05-16

(77) [Semantic Aware Linear Transfer by Recycling Pre-trained Language Models for Cross-lingual Transfer](http://arxiv.org/abs/2505.10945v1), 2025-05-16

(78) [Who You Are Matters: Bridging Topics and Social Roles via LLM-Enhanced Logical Recommendation](http://arxiv.org/abs/2505.10940v1), 2025-05-16

(79) [GenKnowSub: Improving Modularity and Reusability of LLMs through General Knowledge Subtraction](http://arxiv.org/abs/2505.10939v1), 2025-05-16

(80) [Accurate KV Cache Quantization with Outlier Tokens Tracing](http://arxiv.org/abs/2505.10938v1), 2025-05-16

(81) [Connecting the Dots: A Chain-of-Collaboration Prompting Framework for LLM Agents](http://arxiv.org/abs/2505.10936v1), 2025-05-16

(82) [A Dataset for Spatiotemporal-Sensitive POI Question Answering](http://arxiv.org/abs/2505.10928v1), 2025-05-16

(83) [A Survey on the Safety and Security Threats of Computer-Using Agents: JARVIS or Ultron?](http://arxiv.org/abs/2505.10924v1), 2025-05-16

(84) [Vaiage: A Multi-Agent Solution to Personalized Travel Planning](http://arxiv.org/abs/2505.10922v1), 2025-05-16

(85) [A Physics-Informed Convolutional Long Short Term Memory Statistical Model for Fluid Thermodynamics Simulations](http://arxiv.org/abs/2505.10919v1), 2025-05-16

(86) [VISTA: Enhancing Vision-Text Alignment in MLLMs via Cross-Modal Mutual Information Maximization](http://arxiv.org/abs/2505.10917v1), 2025-05-16

(87) [Explain What You Mean: Intent Augmented Knowledge Graph Recommender Built With LLM](http://arxiv.org/abs/2505.10900v1), 2025-05-16

(88) [Multi-Objective Preference Optimization: Improving Human Alignment of Generative Models](http://arxiv.org/abs/2505.10892v1), 2025-05-16

(89) [A Light and Smart Wearable Platform with Multimodal Foundation Model for Enhanced Spatial Reasoning in People with Blindness and Low Vision](http://arxiv.org/abs/2505.10875v1), 2025-05-16

(90) [REI-Bench: Can Embodied Agents Understand Vague Human Instructions in Task Planning?](http://arxiv.org/abs/2505.10872v1), 2025-05-16

(91) [Improve Rule Retrieval and Reasoning with Self-Induction and Relevance ReEstimate](http://arxiv.org/abs/2505.10870v1), 2025-05-16

(92) [Have Multimodal Large Language Models (MLLMs) Really Learned to Tell the Time on Analog Clocks?](http://arxiv.org/abs/2505.10862v1), 2025-05-16

(93) [Improving the Data-efficiency of Reinforcement Learning by Warm-starting with LLM](http://arxiv.org/abs/2505.10861v1), 2025-05-16

(94) [On DeepSeekMoE: Statistical Benefits of Shared Experts and Normalized Sigmoid Gating](http://arxiv.org/abs/2505.10860v1), 2025-05-16

(95) [MatTools: Benchmarking Large Language Models for Materials Science Tools](http://arxiv.org/abs/2505.10852v1), 2025-05-16

(96) [Creativity or Brute Force? Using Brainteasers as a Window into the Problem-Solving Abilities of Large Language Models](http://arxiv.org/abs/2505.10844v1), 2025-05-16

(97) [Alexandria: A Library of Pluralistic Values for Realtime Re-Ranking of Social Media Feeds](http://arxiv.org/abs/2505.10839v1), 2025-05-16

(98) [LARGO: Latent Adversarial Reflection through Gradient Optimization for Jailbreaking LLMs](http://arxiv.org/abs/2505.10838v1), 2025-05-16

(99) [Multimodal Event Detection: Current Approaches and Defining the New Playground through LLMs and VLMs](http://arxiv.org/abs/2505.10836v1), 2025-05-16

(100) [MergeBench: A Benchmark for Merging Domain-Specialized LLMs](http://arxiv.org/abs/2505.10833v1), 2025-05-16
